Algorithm/Number_theory/readme.md

```diff
@@ -34,5 +35,9 @@
 
 <hr>
 
-## Questions :
 
+# Questions related to Number Theory
+ 
+ 1.Factorial of a number ----> [Java](/Code/Java/factorial.java)
```

Comment:
1. numbers are not allowed
2. factorial code is not done by then why have you added that in readme ?